src/wm/title.rs

```diff
@@ -106,15 +106,30 @@ impl Title {
 
     pub fn from_title(title: &str, lang: &str) -> Result<Self, ParseTitleError> {
         let title = title.trim();
-        let lang = lang.trim();
         if title.is_empty() {
             return Err(ParseTitleError::NoTitle);
         }
+        // Wikipedia titles must be less than 256 bytes of UTF-8.
+        // See: https://en.wikipedia.org/wiki/Wikipedia:Naming_conventions_(technical_restrictions)#Title_length
+        if !title.len() < 256 {
+            return Err(ParseTitleError::TitleLong);
```

Comment:
Can larger titles be supported anyway? What will break if this return is skipped?

Reply:
It would show a broken wikipedia link to the user.
But there are other invalid "titles" that are less than 255 characters.